react-main/src/redux/login/actions.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import loginService from "../../services/loginService";
+import { history } from '../store';
+
+export const actions = {
+  GET_TOKEN_SUCCESS: 'GET_TOKEN_SUCCESS',
+  GET_TOKEN_FAILURE: 'GET_TOKEN_FAILURE'
+};
+
+const actionCreators = {
+  login: (credentials) => async dispatch => {
+    //dispatch({ type: actions.GET_BOOKS }); PREGUNTAR
```

Comment:
I'm guessing the question here would be if you need to dispatch that action. Usually you do, because you do stuff there (like setting a `loading: true` to show a spinner), besides it's useful for debugging when using the Redux debug tools